core/domain/classifier/algorithm_registry_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for algorithm registry."""
+
+
+import itertools
+import feconf
+
+from core.domain.classifier import algorithm_registry
+from core.domain.classifier.LDAStringClassifier import LDAStringClassifier
+from core.tests import test_utils
+
+
+class AlgorithmRegistryTest(test_utils.GenericTestBase):
+    """Test the classifier algorithm registry class."""
+
+    def test_get_all_classifier_ids(self):
+        expected_algorithm_ids = [
+            'LDAStringClassifier'
+        ]
+        algorithm_ids = algorithm_registry.Registry.get_all_classifier_ids()
+        self.assertItemsEqual(expected_algorithm_ids, algorithm_ids)
+
+    def test_get_all_classifiers(self):
+        num_classifiers = len(list(itertools.chain(*[
```

Comment:
Isn't this duplicating logic from the registry class? I wonder if we could instead just assert how many classifiers there are without performing this logic. We'd need to update this test as we add/remove classifiers, but that's probably fine.